assets/training/finetune_acft_image/components/pipeline_components/medimage_insight_ft/spec.yaml

```diff
@@ -0,0 +1,141 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: medimage_insight_ft_pipeline
+version: 0.0.1
+type: pipeline
+display_name: Medimage Insight Finetune Pipeline
+description: Pipeline Component to finetune MedImageInsight Model.
+
+inputs:
+  mlflow_embedding_model_path:
+    type: uri_folder
+    optional: false
+    description: Path to the MLflow model to be imported.
+    mode: ro_mount
+
+  eval_image_tsv:
```

Comment:
why TSV and not csv? do you have validations in the execution src for file type?

Reply:
HLS team wants it to be tsv ; It could be csv;